### PR TITLE
Cbusman/fixes to buttons based on platform tools pr

### PR DESCRIPTION
**Pull request by CarolBusman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,7 @@
 [![Bower version][bower-image]][bower-url]
```
Comment (svanherk) [inline README.md]: I think this is undoing all the README changes from yesterday?

File: bower.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "d2l-colors": "^3.1.2",
     "polymer": "Polymer/polymer#1 - 2",
     "d2l-icons": "^5.0.2",
+    "d2l-offscreen": "~3.0.6",
```
Comment (svanherk) [inline bower.json]: You should be good to use the `^3.0.6`, that will only take new minor versions but not major ones.  Were you seeing something different?

Comment (CarolBusman) [reply bower.json]: Nope, just misunderstood how the syntax worked. Fixed.

**Comment by svanherk** [comment]
These socket error hangups are happening a lot with Sauce lately 😒

File: d2l-navigation-button-notification-icon.html [hunk 3]
```diff
@@ -32,22 +33,27 @@
 				display: none;
 				height: 100%;
 				position: absolute;
-				right: calc(-50% + 4px);
```
Comment (svanherk) [inline d2l-navigation-button-notification-icon.html]: Assuming these switches are to adjust for moving from the icon to the div?

File: d2l-navigation-button-notification-icon.html [hunk 4]
```diff
@@ -32,22 +33,27 @@
...
+				top: calc(-50% + 11px);
 				width: 100%;
 			}
-			.d2l-navigation-button-notification-indicator-icon {
-				fill: var(--d2l-color-primary-accent-indicator);
-				height: 100%;
-				width: 100%;
+			:host-context([dir="rtl"]) .d2l-navigation-button-notification-indicator,
+			:host(:dir(rtl)) .d2l-navigation-button-notification-indicator {
+				right: calc(-50% + 12px);
```
Comment (svanherk) [inline d2l-navigation-button-notification-icon.html]: Instead, should we be setting `right: auto` and `left: calc(-50% + 11px);`, so it's the exact opposite of what LTR is?

File: d2l-navigation-button-notification-icon.html (hunk 3 above)
Comment (CarolBusman) [reply d2l-navigation-button-notification-icon.html]: Yup.

File: d2l-navigation-button-notification-icon.html (hunk 4 above)
Comment (svanherk) [reply d2l-navigation-button-notification-icon.html]: Whoops, grabbed the wrong `calc`, but you got what I meant 😛

**Pull request merged**